zoloto/marker_dict.py

```diff
@@ -25,3 +26,7 @@ class MarkerDict(IntEnum):
     DICT_APRILTAG_25H9 = aruco.DICT_APRILTAG_25H9
     DICT_APRILTAG_36H10 = aruco.DICT_APRILTAG_36H10
     DICT_APRILTAG_36H11 = aruco.DICT_APRILTAG_36H11
+
+
+def all_marker_dicts() -> Generator[MarkerDict, None, None]:
```

Comment:
I'm pretty sure that `Enum` is iterable directly?

```python
>>> class Bees(IntEnum):
...     HONEY = 0
...     BUMBLE = 1
...     KILLER = 2
...     CARPENTER = 3
... 
>>> [v for v in Bees]
[<Bees.HONEY: 0>, <Bees.BUMBLE: 1>, <Bees.KILLER: 2>, <Bees.CARPENTER: 3>]
>>> [v.value for v in Bees]
[0, 1, 2, 3]
```

Reply:
waaaaat. Not sure how I forgot about this :man_facepalming: